conf/config-default.yaml

```diff
@@ -167,7 +167,7 @@ nginx_config:                     # config for render the template to generate n
   error_log: logs/error.log
   error_log_level:  warn          # warn,error
   worker_processes: auto          # if you want use multiple cores in container, you can inject the number of cpu as environment variable "APISIX_WORKER_PROCESSES"
-  enable_cpu_affinity: true       # enable cpu affinity, this is just work well only on physical machine
+  enable_cpu_affinity: false       # enable cpu affinity, this is just work well only on physical machine
```

Comment:
```suggestion
  enable_cpu_affinity: false       # disable CPU affinity by default, if APISIX is deployed on a physical machine, it can be enabled and work well.
```

Reply:
> 

Thanks, I have changed